tensorflow/core/api_def/base_api/api_def_CTCLossV2.pbtxt

```diff
@@ -0,0 +1,71 @@
+op {
+  graph_op_name: "CTCLossV2"
```

Comment:
Can you add a line here saying "visibility: HIDDEN"; this will prevent the generation of a tf.ctc_loss_v2 API

Reply:
Sure. Done.